engine/org.eclipse.birt.report.engine.emitter.docx/src/org/eclipse/birt/report/engine/emitter/docx/writer/Footer.java

```diff
@@ -1,15 +1,16 @@
 /*******************************************************************************
- * Copyright (c) 2013 Actuate Corporation.
- * 
+ * Copyright (c) 2013, 2024 Actuate Corporation and others
+ *
  * This program and the accompanying materials are made available under the
  * terms of the Eclipse Public License 2.0 which is available at
  * https://www.eclipse.org/legal/epl-2.0/.
- * 
+ *
  * SPDX-License-Identifier: EPL-2.0
- * 
+ *
  *
  * Contributors:
- *  Actuate Corporation  - initial API and implementation
+ *  Actuate Corporation	- initial API and implementation
+ *  Thomas Gutmann		- add option to handle footer wrapper
```

Comment:
Remember to add this to everything you change in the future.

Reply:
I will remember of it, till now it was not a style restriction to have it at all.
So yes, no problem :o)

Reply:
FYI. This is no longer strictly necessary because the information is in the commit history of git.